### PR TITLE
add arbitrum-one network addresses

### PR DESCRIPTION
**Pull request by adamsoffer**
This PR adds the arbitrum-one network addresses to the subgraph manifest for the Confluence upgrade as specified in the Livepeer.org docs [here](https://github.com/livepeer/docs/blob/yf/deployment/docs/protocol/reference/deployed.md).

### CODE REVIEW

File: networks.yaml [hunk 1]
```diff
@@ -58,6 +58,39 @@ rinkeby:
...
+      address: "289ba1701C2F088cf0faf8B3705246331cB8A839"
+      startBlock: 5856156
+    minterV1:
+      address: "4969dcCF5186e1c49411638fc8A2a020Fdab752E"
+      startBlock: 5856338
+    minter:
+      address: "4969dcCF5186e1c49411638fc8A2a020Fdab752E"
+      startBlock: 5856338
+    ticketBroker:
+      address: "7Beb84c52ce96DFd90431FAA97378994a8baa6df"
```
Comment (yondonfu) [inline networks.yaml:82]: This is the address of the TicketBroker implementation, but we want the address of the TicketBroker proxy at https://arbiscan.io/address/0xa8bB618B1520E284046F3dFc448851A1Ff26e41B

File: networks.yaml [hunk 2]
```diff
@@ -58,6 +58,39 @@ rinkeby:
...
+      startBlock: 5856351
+    pollCreator:
+      address: "8bb50806D60c492c0004DAD5D9627DAA2d9732E6"
+      startBlock: 5857075
+    serviceRegistry:
+      address: "38093CDca43aeCd7bb474983519A246e93A3b0a7"
```
Comment (yondonfu) [inline networks.yaml:88]: This is the address of the ServiceRegistry implementation, but we want the address of the ServiceRegistry proxy at https://arbiscan.io/address/0xC92d3A360b8f9e083bA64DE15d95Cf8180897431

**Review by yondonfu — CHANGES_REQUESTED** [review]

File: networks.yaml (hunk 1 above)
Comment (adamsoffer) [reply networks.yaml]: ah nice spot. Pushed up fix.

File: networks.yaml (hunk 2 above)
Comment (adamsoffer) [reply networks.yaml]: fixed

**Review by yondonfu — APPROVED** [review]
LGTM!

**Comment by adamsoffer** [comment]
@yondonfu fixed addresses and redeployed

**Pull request merged**